packages/compartment-mapper/test/fixtures-cjs-compat/node_modules/defineprop/main.js

```diff
@@ -0,0 +1,18 @@
+// fragments of logic adapted from ethereumjs-util@7.1.0
+"use strict";
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+    if (k2 === undefined) k2 = k;
+    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
+}) : (function(o, m, k, k2) {
+    if (k2 === undefined) k2 = k;
+    o[k2] = m[k];
+}));
+var __exportStar = (this && this.__exportStar) || function(m, exports) {
+    for (var p in m) if (p !== "default" && !exports.hasOwnProperty(p)) __createBinding(exports, m, p);
+};
+Object.defineProperty(exports, "__esModule", { value: true });
+/// <reference path="./@types/ethjs-util/index.ts"/>
+
+const stuff = { stuff: 1 };
+__exportStar(stuff, exports);
```

Comment:
Should have some assertion on the import side.

Reply:
:+1: